distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -143,8 +143,14 @@ case "$1" in
 		exit 0
 	fi
 
-	# Prepare environment
-	mkdir -p "$LOG_DIR" "$DATA_DIR" && chown "$ES_USER":"$ES_GROUP" "$LOG_DIR" "$DATA_DIR"
+  # Prepare environment
+  # Check $DATA_DIR for a comma
+  if [ test "${DATA_DIR#*,}" != "$DATA_DIR" ]; then
```

Comment:
I get `[: too many arguments` when I run this in bash locally. Do you need the `test` bit? It works without that part.